tensorboard/webapp/runs/views/runs_table/regex_edit_dialog.ng.html

```diff
@@ -19,12 +19,22 @@ <h1 mat-dialog-title>Color runs by regex</h1>
 
   <mat-dialog-content>
     <p>Enter a regex with capturing groups to match against run names:</p>
+    <mat-form-field>
+      <mat-label>Regex type</mat-label>
+      <mat-select
+        [(value)]="regexMatchType"
+        (selectionChange)="regexTypeChange($event)"
+      >
+        <mat-option value="regex_by_run">Run Name</mat-option>
```

Comment:
These string literals are repeated many times. Let's define them as instance variables in the component class (we can try making them static as well, but I'm not sure if you'd be able to use them in the template)